<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>IndentGui</class>
 <widget class="QMainWindow" name="IndentGui" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>UniversalIndentGUI</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="Icons.qrc" >:/new/prefix1/icon2.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QPushButton" name="pbOpenFile" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:MS Shell Dlg; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt;">Opens a dialog for selecting a source code file.&lt;/span>&lt;/p>&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">This file will be used to show what the indent tool changes.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="text" >
         <string>Open File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbLoadIndentCfg" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:MS Shell Dlg; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Loads the configuration file for an indent tool.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="text" >
         <string> Load Indent Cfg </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbLivePreview" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:MS Shell Dlg; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">Turns the preview of the reformatted source code on and off.&lt;/p>&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">In other words it switches between formatted and nonformatted code.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="text" >
         <string>Live Indent Preview</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbHighlight" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:MS Shell Dlg; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Enables and disables the highlightning of the source&lt;/p>&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">code shown below. (Still needs some performance improvements)&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="text" >
         <string>Syntax Highlight</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbExit" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:MS Shell Dlg; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Quits the UniversalIndentGUI&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="text" >
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="txtedLineNumbers" >
        <property name="maximumSize" >
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font" >
         <font>
          <family>Courier New</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="contextMenuPolicy" >
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="verticalScrollBarPolicy" >
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="lineWrapMode" >
         <enum>QTextEdit::NoWrap</enum>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="txtedSourceCode" >
        <property name="font" >
         <font>
          <family>Courier New</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="lineWrapMode" >
         <enum>QTextEdit::NoWrap</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBooah" >
    <property name="title" >
     <string>View</string>
    </property>
    <addaction name="actionZoom" />
   </widget>
   <widget class="QMenu" name="menuTest" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionAnot_her" />
    <addaction name="actionOpen" />
    <addaction name="actionSave" />
    <addaction name="separator" />
    <addaction name="actionPrint" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <addaction name="menuTest" />
   <addaction name="menuBooah" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>581</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>1</hsizetype>
     <vsizetype>7</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="contextMenuPolicy" >
    <enum>Qt::ActionsContextMenu</enum>
   </property>
   <property name="movable" >
    <bool>false</bool>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionOpen" />
   <addaction name="separator" />
   <addaction name="actionExit" />
  </widget>
  <action name="actionAnot_her" >
   <property name="text" >
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="icon" >
    <iconset resource="Icons.qrc" >:/new/prefix1/icon3.png</iconset>
   </property>
   <property name="text" >
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="text" >
    <string>Save</string>
   </property>
  </action>
  <action name="actionPrint" >
   <property name="text" >
    <string>Print</string>
   </property>
  </action>
  <action name="actionZoom" >
   <property name="text" >
    <string>Zoom</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="text" >
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="Icons" />
  <include location="Icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>pbExit</sender>
   <signal>clicked()</signal>
   <receiver>IndentGui</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>793</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel" >
     <x>554</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>IndentGui</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>403</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
